#!/usr/bin/env python
PACKAGE = "kinect_slam"
 
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_signature_threshold", int_t, 0, "max_signature_threshold", 1, 10, 300)
gen.add("match_threshold", double_t, 0, "match_threshold", 0.85, 0, 1)
gen.add("new_landmark_threshold", int_t, 0, "new_landmark_threshold", 500, 0, 5000)

exit(gen.generate(PACKAGE, "kinect_slam", "EKFSLAM"))
